app/scripts/vendors/phaser/phaser.js

```diff
@@ -33466,7 +33466,7 @@ Phaser.Group.prototype.set = function (child, key, value, checkAlive, checkVisib
 *
 * @method Phaser.Group#setAll
 * @param {string} key - The property, as a string, to be set. For example: 'body.velocity.x'
-* @param {any} value - The value that will be set.
+* @param {*} value - The value that will be set.

```

Comment:
No jak można było tak zepsuć JSDoca... to ja nie wiem... 
Musiałem to poprawić! :smiling_imp: